NuRadioMC/SignalProp/analyticraytracing.py

```diff
@@ -2198,6 +2597,51 @@ def apply_propagation_effects(self, efield, i_solution):
         if self._config['propagation']['focusing']:
             focusing = self.get_focusing(i_solution, limit=float(self._config['propagation']['focusing_limit']))
             spec[1:] *= focusing
+        
+        # apply the birefringence effect
+        if self._config['propagation']['birefringence']:
+            
+            if "birefringence_model" in self._config['propagation']:
```

Comment:
I don't understand this if statement. Shouldn't there always be a default birefringence model be set? If the user hasn't specified it, then it the one from the `default_config.yaml` should be taken. 
This code implicitly sets a default birefringence model. If a birefringence model is defined in `default_config.py`, then it is not necessary to check if it exists in the config and this code can be simplified.

Reply:
I wasn't sure about the best way to implement this so it's quite possible this can be done in a better way. The idea was to have a running system even if the user only sets `birefringence = True` . However, as you say the code can be simplified a lot if we assume there will always be a  `birefringence_model` and a `birefringence_propagation` set in the config file because I added them to `default_config.yaml`.

Reply:
you can safely assume that if you added it to the default_config.yaml

Reply:
Done